### DISCUSSION TITLE
Getting an error when creating a table in nextjs

### DISCUSSION THREAD

**Discussion by LNFWebsite** [vercel/next.js#35795] [Help]
Hi, I'm getting this error:

    Error: Hydration failed because the initial UI does not match what was rendered on the server.

With this code alone:

```
import Head from 'next/head'
import styles from '../styles/Home.module.css'

export default function Home() {
  return (
    <table>t</table>
  )
}
```

When I leave the table completely empty with nothing in it, there's no error. But upon adding literally anything including a `<tr></tr>` the hydration error appears. This occurs the same when the table is in a `<div>` container rather than being returned directly.

This is a brand new project created with npx create-next-app@latest that I didn't reconfigure at all.

**Comment by peterleiva** [comment]


**Reply by LNFWebsite** [reply]
Is there any solution at the moment?

**Comment by HeinZarNe** [comment]
hello do you get solution??

**Comment by joker63** [comment]
If you add an empty tbody/thead tag, there will be no error
`<table>
					<thead>
					</thead>
				</table>`

**Comment by icyJoseph** [comment]
Hi,

We did find a solution for this back in https://github.com/vercel/next.js/discussions/36754, it even ended up on an issue on React's repo.

Long story short, any form of invalid DOM nesting causes React to throw on the server, and that ends up as mismatched HTML between server and client.

It was also a problem on React 17, and before, but it never exploded like this because the of the rendering root. Nowadays, this produces a double error (first one is a warning/error about DOM nesting), in which the latter is fatal.

**Comment by httol** [comment]
if you use tag not belong to table among standard table tag. <div></div>
